themes/default/content/docs/install/_index.md

```diff
@@ -555,6 +555,48 @@ You can specify a specific version with [Chocolatey package manager](https://cho
 
 {{% /chooser %}}
 
+In addition to insalling a specific version, the latest dev version can also be installed automatically.  This version contains the latest changes that have been merged to the main branch.
```

Comment:
It's probably worth having a new header for this section, perhaps something like `## Installing dev releases` or something like that